.travis.yml

```diff
@@ -18,9 +18,12 @@ install:
 - pip install --no-cache-dir . -r dev-requirements.txt
 - npm install
 - npm run webpack
+# sphinx requirements
+- pip install --no-cache-dir . -r docs/doc-requirements.txt
```

Comment:
We should remove the `--no-cache` here. That's only for docker. There's also no `docs/doc-requirements.txt`, but there is a `docs/environment.yml`.

Reply:
but there is `doc/doc-requirements.txt`, fixed that typo.

Reply:
I find it annoying/confusing that we have both in the docs. It always leaves me guessing which should I use, who uses which file etc :_/